src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -227,14 +227,16 @@ export default defineComponent({
     },
     /**
      * Determines which `v-calendar` type to initialize.
-     * Three of the values (`date`, `time`, `dateTime`) are passed verbatim to `v-calendar`,
-     * whereas `relative` denotes a component instance made up solely of time frames.
+     * - { `date`, `time`, `dateTime` } are passed verbatim to `v-calendar`,
+     * - `relativeOnly` denotes a component instance made up solely of time frames
+     * - `relativeDateTime` relative time frames + datetime calendar
+     * - `relativeDate` relative time frames + date calendar
      */
     mode: {
       type: String,
       required: true,
       validator: (value: string): boolean => {
-        return ['relative', 'date', 'time', 'dateTime'].includes(value)
+        return ['relativeOnly', 'relativeDateTime', 'relativeDate', 'date', 'time', 'dateTime'].includes(value)
```

Comment:
What is `relativeOnly`? If it's just time, then maybe `relativeTime` instead?

Reply:
`relativeOnly` literally only shows the time frame buttons

I'll rename the other two to be `relativeAndCustomDate` | `relativeAndCustomDateTime` to be ultra specific :)

Reply:
TBH that seems just as confusing, if not more. Can we be more concise somehow?

Reply:
settled on 

```
date
time
dateTime
relative
relativeDate
relativeDateTime
```